### PR TITLE
Feat: Add OSIF to description

### PR DESCRIPTION
**Pull request by SukkaW**

### CODE REVIEW

File: 04-操作系统补丁/SSDT-OC-XOSI.dsl [hunk 1]
```diff
@@ -3,6 +3,10 @@
 // Find:     4F534944
 // Replace:  58534944
 //
+// In config ACPI, OSID to XSID
```
Comment (athlonreg) [inline 04-操作系统补丁/SSDT-OC-XOSI.dsl]: Hi, this looks good for me! But is this line should be OSIF to XSIF?

File: 04-操作系统补丁/SSDT-OC-XOSI.md [hunk 2]
```diff
@@ -4,6 +4,10 @@
 // Find:     4F534944
 // Replace:  58534944
 //
+// In config ACPI, OSID to XSID
```
Comment (athlonreg) [inline 04-操作系统补丁/SSDT-OC-XOSI.md]: Same as above.

File: 04-操作系统补丁/SSDT-OC-XOSI.dsl (hunk 1 above)
Comment (SukkaW) [reply 04-操作系统补丁/SSDT-OC-XOSI.dsl]: Yeah, a typo error!

**Comment by SukkaW** [comment]
@athlonreg Typos fixed and rebased.

**Review by williambj1 — APPROVED** [review]

**Pull request merged**